x/httpproxy/connect_handler.go

```diff
@@ -119,7 +119,16 @@ func (h *connectHandler) ServeHTTP(proxyResp http.ResponseWriter, proxyReq *http
 
 	// Relay data between client and target in both directions.
 	go func() {
-		io.Copy(targetConn, clientRW)
+		// io.Copy prefers WriteTo, which clientRW implements. However,
+		// bufio.ReadWriter.ReadFrom issues an empty Write() call, which flushes
```

Comment:
Do you mean `bufio.ReadWriter.WriteTo` here?

Reply:
Yes. Fixed.